galerautils/src/gu_mutex.c

```diff
@@ -21,6 +21,7 @@ gu_mutex_init = { .target_mutex      = PTHREAD_MUTEX_INITIALIZER,
                   .file              = __FILE__,
                   .line              = __LINE__
 };
+#endif
```

Comment:
Would it be better to delete the definition altogether if it is not used anywhere?